src/schemas.py

```
@@ -84,27 +100,46 @@
+class SessionPeerConfig(BaseModel):
+    observe_others: bool = Field(
+        default=False,
+        description="Whether this peer should form a session-level theory-of-mind representation of other peers in the session",
+    )
+    observe_me: bool = Field(
+        default=True,
+        description="Whether other peers in this session should try to form a session-level theory-of-mind representation of this peer",
+    )
+
+
 class SessionCreate(SessionBase):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)